scripts/arangodb/start-arango-db.sh

```diff
@@ -12,13 +38,18 @@ set -e
 echo "echo stop & remove old docker [$ARANGO_DB_SERVER] and starting new fresh instance of [$ARANGO_DB_SERVER]"
 (docker kill $ARANGO_DB_SERVER || :) && \
   (docker rm $ARANGO_DB_SERVER || :) && \
-  docker run -e ARANGO_ROOT_PASSWORD=$ARANGO_DB_ROOT_PASSWORD --name $ARANGO_DB_SERVER -p $ARANGO_DB_PORT:8529 -d arangodb
+  docker run -e ARANGO_ROOT_PASSWORD=$ARANGO_DB_ROOT_PASSWORD \
+  -e ARANGO_TEST_USER_PASSWORD=$ARANGO_DB_USER_PASSWORD \
+  --name $ARANGO_DB_SERVER -p $ARANGO_DB_PORT:8529 -d$ARANGO_VOLUME_PATH_CMD \
+  arangodb
 
 # wait for pg to start
 echo "sleep wait for arango-db-server [$ARANGO_DB_SERVER] to start";
 sleep 3;
 
 # TODO
-# create the db 
+# create the db
 # set environment variables
-# create new db through arangosh 
\ No newline at end of file
+# create new db through arangosh
+
+docker exec -it $ARANGO_DB_SERVER $ARANGO_DOCKER_VOLUME_DESTINATION/arango-setup.sh
```

Comment:
Is there  a way to have a fallback in case `$ARANGO_DOCKER_VOLUME_DESTINATION`  is empty? I just want to be sure we don't mount the repo directory somehow.